bot/cogs/todo_list.py

```diff
@@ -0,0 +1,161 @@
+                    r = some_todos.split(",")
+                    for td in r:
+                        if td == "":
+                            continue
+                        else:
+                            self.del_the_todo.append(f"{td.strip(' ').rstrip(' ')}\n")
+
+                    with open(f"{BASE_DIR}/save_data_td.JSON", "r+") as read:
+                        f = json.load(read)
+                        new = [
+                            x
+                            for x in f[str(ctx.author.id)]
+                            if x not in self.del_the_todo
+                        ]
+                        removed = [
+                            x for x in f[str(ctx.author.id)] if x in self.del_the_todo
+                        ]
+                        f[str(ctx.author.id)] = new
+                        with open(f"{BASE_DIR}/save_data_td.JSON", "w+") as save:
+                            json.dump(f, save)
+                            completion_message = (
+                                f"{ctx.author.mention}, your todo list after deletion:\n{''.join(new)}\n"
+                                f"Removed todos:\n{''.join(removed)}"
+                            )
+                            await ctx.send(completion_message)
+                else:
+                    completion_message = (
+                        f"{ctx.author.mention}, your todo list entry is empty. "
+                        f"Please enter a todo list format (i.e `.del_todos Make assignments, Learn Python`) with comma "
+                        f"on "
+                        f"every todo."
+                    )
+                    await ctx.send(completion_message)
+
+        create_todo_list.start()
+
+    @command(brief="A simple todolist in text")
+    async def todolist(self, ctx, *, some_todos=None):
+        """Creates a todolist for the user."""
+
+        if ctx.author.id not in self.todos:
+            self.todos[ctx.author.id] = 0
+        if self.todos[ctx.author.id] < 1:
+            await self.todos_wrapper(
+                ctx=ctx, task_type="todolist", some_todos=some_todos
+            )
+
+    @command(brief="Deletes some todos from a todolist")
+    async def del_todos(self, ctx, *, some_todos=None):
+        """Deletes todos from a todolist for the user."""
+
+        if ctx.author.id not in self.del_todos:
+            self.del_todos[ctx.author.id] = 0
+        if self.del_todos[ctx.author.id] < 1:
+            await self.todos_wrapper(
+                ctx=ctx, task_type="del_todos", some_todos=some_todos
+            )
+
+    @command(brief="Checks your current todolist")
+    async def check_todos(self, ctx):
+        """Reads json and shows current todolist of user."""
+
+        with open(f"{BASE_DIR}/save_data_td.JSON", "r+") as read:
+            file = json.load(read)
+            try:
+                await ctx.send(
+                    f"Your current todolist:\n{''.join(file[str(ctx.author.id)])}"
```

Comment:
This should be an embed. It should also enumerate so that the Todo items are numbered.